topology/graph/server.go

```diff
@@ -41,11 +41,12 @@ const (
 )
 
 type Server struct {
-	Graph    *Graph
-	Alert    *Alert
-	Router   *mux.Router
-	wsServer *WSServer
-	Host     string
+	Graph     *Graph
+	Alert     *Alert
+	Router    *mux.Router
+	wsServer  *WSServer
+	Host      string
+	waitGroup *sync.WaitGroup
```

Comment:
be consistant in name : wg vs waitGroup
Same comment here declare it like : wg sync.WaitGroup